actiontext/lib/action_text/content.rb

```diff
@@ -104,7 +104,9 @@ def inspect
     end
 
     def ==(other)
-      if other.is_a?(self.class)
+      if (klass = self.class) == other.class
```

Comment:
Any particular reason to assign `klass = self.class` here?

Reply:
No, just to avoid calling `self.class` twice. I have no idea if instantiating an object is cheaper. I went ahead and removed it to make things more readable.